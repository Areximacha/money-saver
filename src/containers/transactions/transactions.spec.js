import React from 'react'
import renderer from 'react-test-renderer'
import AccountContext from '../../context/accountProvider'

import Transactions from './transactions'

const mockState = {
  transactions: [],
}

const renderWithContext = node =>
  renderer.create(
    <AccountContext.Provider value={mockState}>{node}</AccountContext.Provider>
  )

describe('<Transactions/>', () => {
  describe('@renders', () => {
    it('renders correctly', () => {
      const tree = renderWithContext(<Transactions />).toJSON()

      expect(tree).toMatchSnapshot()
    })
  })
})
