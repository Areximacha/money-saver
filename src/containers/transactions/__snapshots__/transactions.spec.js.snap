// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Transactions/> @renders renders correctly 1`] = `
<div
  className="css-9dosor-TransactionsContainer eegxj020"
>
  <h3>
    Your Outgoing Transactions This Week
  </h3>
  <div
    className="css-1gbizu7-TransactionsBox eegxj021"
  />
</div>
`;
